composer.json

```
@@ -4,42 +4,48 @@
   "type": "drupal-custom-profile",
   "license": "GPL-2.0+",
   "minimum-stability": "dev",
-  "require": {
-    "composer/installers": "^1.2",
-    "drupal/address": "~1.2",
-    "drupal/admin_toolbar": "~1.20",
-    "drupal/allowed_formats": "~1.1",
-    "drupal/chosen": "~2.4",
-    "drupal/components": "~1.0",
-    "drupal/config_readonly": "~1.0@beta",
-    "drupal/core": "~8.6",
-    "drupal/default_content": "^1.0@alpha",
-    "drupal/diff": "^1.0@RC",
-    "drupal/ds": "~3.3",
-    "drupal/google_analytics": "~2.1",
-    "drupal/honeypot": "^1.27",
-    "drupal/menu_block": "^1.5",
-    "drupal/menu_position": "~1.0@alpha",
-    "drupal/metatag": "~1.3",
-    "drupal/pathauto": "~1.0",
-    "drupal/redirect": "~1.0-beta1",
-    "drupal/role_delegation": "^1.0@beta",
-    "drupal/rules": "^3.0@alpha",
-    "drupal/ui_patterns": "~1.0",
-    "drupal/views_block_filter_block": "^1.0@beta",
-    "drupal/views_bulk_operations": "~2.4",
-    "drupal/view_unpublished": "^1.0@alpha",
-    "drupal/webform": "~5.0@beta",
-    "drupal/xmlsitemap": "~1.0@alpha",
-
-    "su-sws/stanford_basic": "dev-8.x-4.x",
-    "su-sws/stanford_date_formats": "dev-8.x-1.x",
-    "su-sws/stanford_image_styles": "dev-8.x-1.x",
-    "su-sws/stanford_media": "dev-8.x-1.x",
-    "su-sws/stanford_page": "dev-8.x-2.x",
-    "su-sws/stanford_paragraph_types": "dev-8.x-2.x",
-    "su-sws/stanford_ssp": "dev-8.x-1.x",
-    "su-sws/stanford_text_editor": "dev-8.x-1.x",
-    "su-sws/jumpstart_ui": "dev-8.x-1.x"
+    "repositories": [
+        {
+            "type": "composer",
+            "url": "https://packages.drupal.org/8"
+        }
+    ],
+    "require": {
+        "composer/installers": "^1.2",
+        "drupal/address": "~1.2",
+        "drupal/admin_toolbar": "~1.20",
+        "drupal/allowed_formats": "~1.1",
+        "drupal/chosen": "~2.4",
+        "drupal/components": "~1.0",
+        "drupal/config_readonly": "~1.0@beta",
+        "drupal/core": "~8.6",
+        "drupal/default_content": "^1.0@alpha",
+        "drupal/diff": "^1.0@RC",
+        "drupal/ds": "~3.3",
+        "drupal/google_analytics": "~2.1",
+        "drupal/honeypot": "^1.27",
+        "drupal/imagemagick": "^3.0",
+        "drupal/menu_block": "^1.5",
+        "drupal/menu_position": "~1.0@alpha",
+        "drupal/metatag": "~1.3",
+        "drupal/pathauto": "~1.0",
+        "drupal/redirect": "~1.0-beta1",
+        "drupal/role_delegation": "^1.0@beta",
+        "drupal/rules": "^3.0@alpha",
+        "drupal/ui_patterns": "~1.0",
+        "drupal/view_unpublished": "^1.0@alpha",
+        "drupal/views_block_filter_block": "^1.0@beta",
+        "drupal/views_bulk_operations": "~2.4",
+        "drupal/webform": "~5.0@beta",
+        "drupal/xmlsitemap": "~1.0@alpha",
```

```suggestion
        "drupal/xmlsitemap": "~1.0@alpha",
        "ext-imagick": "*",
```

RE: http://syrwebdev.com/composer/2015/01/07/using-composer-to-require-php-modules.html
> Composer has platform packages, which are virtual packages for things that are installed on the system but are not actually installable by Composer. This includes PHP itself, PHP extensions and some system libraries.

---

ahh thats a good thing to include in there.